src/components/expression/expression-renderer.js

```diff
@@ -616,7 +796,7 @@ genotet.ExpressionRenderer.prototype.highlightHoverCell_ = function(cell) {
 
 /**
  * Unhighlights the hover cell for the heatmap.
- * @param {*} cell TODO(liana): fill the type.
+ * @param {!Object} cell
```

Comment:
Umm this {!Object} type is not very informative. You may want to declare your cell type
```js
/**
 * @typedef {{
 *   container_: ?jQuery,
 *   geneName: ?string,
 *   conditionName: ?string,
 *   row: number,
 *   column: number,
 *   value: number,
 *   colorscaleValue: ?number
 * }}
 */
genotet.ExpressionRenderer.Cell;
```
https://developers.google.com/closure/compiler/docs/js-for-compiler#tag-typedef